<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>771</width>
     <height>510</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string>img</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_output">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>551</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_input">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>551</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab3</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>761</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>420</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>串口选择</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>440</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>COM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>COM8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>440</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>440</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QLabel" name="Light">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_load">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>加载设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>添加一行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_apply">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
